src/vos/evtree.c

```diff
@@ -3462,7 +3462,7 @@ evt_node_delete(struct evt_context *tcx)
 {
 	struct evt_trace	*trace;
 	struct evt_node		*node;
-	struct evt_node_entry	*ne;
+	struct evt_node_entry	*ne = NULL;
```

Comment:
For whatever reason, gcc now thinks this may not be initialized (when it is), possibly something to do with not calling assert() with the condition anymore.